cfgov/unprocessed/js/organisms/MortgagePerformanceTrends/utils.js

```diff
@@ -1,16 +1,20 @@
-import ajax from 'xdr';
+const BASE_URL = '/data-research/mortgages/api/v1/metadata/';
+const cache = {};
 
-const COUNTIES_URL =
-  '/data-research/mortgages/api/v1/metadata/state_county_meta';
-const METROS_URL = '/data-research/mortgages/api/v1/metadata/state_msa_meta';
-const NON_METROS_URL = '/data-research/mortgages/api/v1/metadata/non_msa_fips';
-const STATES_URL = '/data-research/mortgages/api/v1/metadata/state_meta';
-let counties;
-let metros;
-let nonMetros;
-let states;
 let globalZoomLevel = 10;
 
+/**
+ * @param {string} dataType - dataType to append to the BASE_URL
+ * @param {Function} cb - The callback to pass the data
+ * @returns {undefined} The result of calling the callback on the resolved data
```

Comment:
Trivial: Returns a `{Promise}`, no?

Reply:
Oh I see it has two return types. So I think `{*|Promise}` or `{*}` would be more correct.

Reply:
I just changed it to always return a promise in a90ecdc :)

Reply:
luv it